server/api/controllers/forms/get-submission.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+  friendlyName: "Retrieve Submissions for a certain form",
+
+  description: "Create a new submission entry from form data",
+
+  inputs: {
+    formId: {
+      type: "string",
+      required: true,
+    },
+    timeRange: {
```

Comment:
maybe can add an example of a timeRange so that its more clear what the structure should look like